pkg/runtime/wfengine/backends/actors/workflow_actor.go

```diff
@@ -598,12 +598,21 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 			if errMarshal != nil {
 				return errMarshal
 			}
-
+			requestBytes := eventData
```

Comment:
Is this not backwards-incompatible? If a sidecar that is running 1.12 receives this message, it may not know how to parse this?

Reply:
I think not. If a sidecar is running 1.12, here we will be sending only `startEvent` as rawData: https://github.com/dapr/dapr/blob/release-1.12/pkg/runtime/wfengine/workflow.go#L485

and then we would be parsing that only in `CreateWorkflowInstanceMethod`: https://github.com/dapr/dapr/blob/986d38b8d0524753bb5331a3349e11f17200d3ca/pkg/runtime/wfengine/workflow.go#L170

The issue here is PR #7308 added the struct here: https://github.com/dapr/dapr/blob/ded0ace346ebb129ac319b961dd05516bc0b098d/pkg/runtime/wfengine/backends/actors/workflow_actor.go#L180 but missed to add here and so the inconsistency causing the issue.

Reply:
So #7308 is not backwards-compatible?

Reply:
I meant #7308 is backwards compatible. It's just that it introduced a bug and the current PR fixes it. But it should worknwell even if the sidecar is from 1.12.